imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -36,6 +36,17 @@ energy_attrs:
     VALIDMAX: 127
     VAR_TYPE: support_data
 
+spin_sector_attrs:
+    <<: *default
+    CATDESC: Spin sector indicating range of spin angles
+    FIELDNAM: Spin sector
+    FORMAT: I4
+    LABLAXIS: spin sector
+    UNITS: ' '
+    VALIDMIN: 0
+    VALIDMAX: 1152
+    VAR_TYPE: support_data
```

Comment:
I have plans for a future PR that updates these yaml files to adhere to the proper nomenclature (https://lasp.colorado.edu/galaxy/display/IMAP/IMAP+Nomenclature+of+Science+Data+Products), so this may change, but works for now.